e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/types/ConstraintRecord.java

```diff
@@ -14,7 +15,7 @@ public record ConstraintRecord(
   public static ConstraintRecord fromJSON(JsonObject json){
     return new ConstraintRecord(
         json.getBoolean("success"),
-        json.getJsonObject("results").isEmpty() ? null : ConstraintResult.fromJSON(json.getJsonObject("results")),
+        json.getJsonObject("results").isEmpty() ? new ConstraintResult(-1,"","",new ArrayList<>(), new ArrayList<>(),new ArrayList<>()) : ConstraintResult.fromJSON(json.getJsonObject("results")),
```

Comment:
Minor: This change should be reverted, as none of this information is in the JSON object

Reply:
got it, reverted

Reply:
No worries, I have updated all the test and added this check:

```java
    final var constraintResponse = constraintsResponses.get(0);
    assertTrue(constraintResponse.success());
    assertTrue(constraintResponse.result().isPresent());
    final var constraintResult = constraintResponse.result().get();
```